rosa-hypershift/README.md

```diff
@@ -0,0 +1,167 @@
+
+After the clusters are successfully installed, the wrapper will execute e2e-benchmarking scripts to assess the performance of the clusters. The benchmarking results will be documented and stored in the corresponding Elasticsearch URL (ES_URL).
+```
+$ python rosa-hypershift/rosa-hosted-wrapper.py --cluster-name-seed ocp413 --cluster-count 5 --aws-account-file <PATH>/.aws/credentials --aws-profile default --rosa-cli <PATH_TO>/rosa --ocm-token <OCM_TOKEN> --workers 3 --es-url https://<USERNAME>:<PASSWORD>@<ES_URL>:443 --es-index hypershift-wrapper-timers --es-insecure --log-level DEBUG --log-file /tmp/ocp413.log --batch-size 1 --delay-between-batch 20 --cluster-load-duration 15m --cluster-load-jobs-per-worker 9 --cluster-load-job-variation 0 --wildcard-options "--version 4.12.15" --create-vpc --terraform-cli <PATH_TO>/terraform --manually-cleanup-secgroups --workers-wait-time 60 --delay-between-cleanup 30 --add-cluster-load --common-operator-roles  --rosa-env staging  --provision-shard <SC_PROVISION_SHARD>
+```
+
+
+## Required packages
+
+- python3
+- pip3
+
+A requirements.txt file contains the required python-pip packages and can be installed via:
+
+```
+$ pip3 install -r requirments.txt
+```
+
+## Required Variables
+
+The wrapper takes the following required variables:
+
+| Option | Description | Default |
+|--------|-------------|---------|
+| --cluster-name-seed| Seed used to generate cluster names. 6 chars max | perfc |
+| --ocm-token | Obtain offline via on https://cloud.redhat.com/openshift/token | -- |
+| --cluster-count | Total number of clusters to create | -- |
+| --aws-account-file | AWS account file to use | -- |
+| --aws-profile | AWS profile to use if more than one are present on aws config file | -- |
+| --workers | Number of workers for the hosted cluster (min: 3). If list (comma separated), iteration over the list until reach number of clusters | 3 |
+
+
+---
+## Elasticsearch variables:
+
+**NOTE**: If Elasticsearch server and port are omitted then the cluster test will run as normal without uploading any information
+
+| Option | Description | Default |
+|--------|-------------|---------|
+| --es-url | The elasticsearch server URL, including protocol and port (if required)| -- |
+| --es-insecure | If ES is setup with ssl, but can disable tls cert verification | False |
+| --es-index | The index to write to. | hypershift-wrapper-timers |
+| --es-index-retry | Number of retries to connect to ES | 5 |
+| --es-index-only | Upload all metadata.json files found under PATH to elasticsearch | -- |
+
+## Optional variables:
+
+Optional variables are parameters that offer flexibility and allow users to customize the behavior or configuration of the program based on their specific needs.
+
+| Option | Description | Default |
```

Comment:
No that we have these many arguments being passed in the CLI, can we simply use configuration files to do that? Open to other thoughts too.

Reply:
> Now that we have these many arguments being passed in the CLI, instead can we simply use configuration files to do that? Open to other thoughts too.

Yes, some are important and the rest are "optional" parameters, would be good to capture all values into a default conf. file and supply the rest via cli

But should be tackled outside the scope of this PR thinking via an RFE.